examples/basic_task.py

```diff
@@ -0,0 +1,30 @@
+import discord
+from discord.ext import tasks
+
+
+# A task that sends a message to the channel once per minute.
+@tasks.loop(minutes=1)
+async def my_task(client):
+    # https://discordpy.readthedocs.io/en/latest/api.html?highlight=wait%20until%20re#discord.Client.wait_until_ready
```

Comment:
Why not use the shorter link? https://discordpy.rtfd.io/en/latest/api.html#discord.Client.wait_until_ready